test/python/scenarios.json

```diff
@@ -194,20 +238,24 @@
             }
         }
     },
-    "install_alternate_tools_rhel_family": {
-        "image": "almalinux:8",
+    "install_alternate_tools_alpine": {
+        "image": "alpine:3",
         "features": {
+            "common-utils": {
+                "username": "vscode"
+            },
             "python": {
                 "version": "3.12",
                 "toolsToInstall": "ruff,mypy,bandit,pipenv,virtualenv,pytest"
             }
         }
     },
-    "centos-7": {
```

Comment:
Curious, why was this removed?

Reply:
Maybe I misunderstood; I thought you requested this in an earlier comment: https://github.com/devcontainers/features/pull/1015#discussion_r1728054964

Shall I revert the change?

Reply:
I'm going to revert this change, since I haven't heard anything back